### PR TITLE
d formulae: add `head` branch

### PR DESCRIPTION
**Pull request by chenrui333**
update formulae head branches

relates to Homebrew/brew#11720

### CODE REVIEW

File: Formula/dwm.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Dwm < Formula
   sha256 "97902e2e007aaeaa3c6e3bed1f81785b817b7413947f1db1d3b62b8da4cd110e"
   license "MIT"
   revision 2
-  head "https://git.suckless.org/dwm", using: :git
+  head "https://git.suckless.org/dwm.git", branch: "master"
```
Comment (SMillerDev) [inline Formula/dwm.rb:8]: This does not work

File: Formula/dmenu.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class Dmenu < Formula
   sha256 "fe18e142c4dbcf71ba5757dbbdea93b1c67d58fc206fc116664f4336deef6ed3"
   license "MIT"
   revision 1
-  head "https://git.suckless.org/dmenu/", using: :git
+  head "https://git.suckless.org/dmenu.git", branch: "master"
```
Comment (SMillerDev) [inline Formula/dmenu.rb:8]: This does not work

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/dwm.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/dwm.rb]: Fixed the issue.

File: Formula/dmenu.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/dmenu.rb]: Fixed the issue.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**